src/util.js

```diff
@@ -57,3 +59,20 @@ export function setByPath (obj, path, value) {
 
 	return parent && (parent[lastKey] = value);
 }
+
+const message = "Node has no parent pointers set. Call setParent() on the node or updateParents() on an ancestor to add parent pointers to this node";
+
+/**
+ * Check a node to see if it has parent pointers set
+ * @param {Node} node the node to check for parents
+ * @param {"error" | "warn"} severity the severity of the issue
+ * @throws {Error} if the severity is "error" and the node has no parent
+ */
+export function checkParentPointers (node, severity = "error") {
+	if (!getParent(node)) {
```

Comment:
The parent being `null` is not an error, it's just the root of the tree

Reply:
Yes, force of habit using `!`